ws-nextjs-app/pages/[service]/new_live/[id]/Posts/index.tsx

```diff
@@ -1,4 +1,6 @@
-import React from 'react';
+/** @jsx jsx */
```

Comment:
Does this also need the classicRuntime? (Had you already defaulted that for the NextJS app?)

Reply:
Don't think we will need this now as mentioned below.